cluster/saltbase/salt/kubelet/default

```diff
@@ -41,6 +41,12 @@
 {% endif -%}
 
 {% set config = "--config=/etc/kubernetes/manifests" -%}
+
+{% set manifest_url = "" -%}
+{% if grains['roles'][0] == 'kubernetes-master' and grains.cloud in ['gce'] -%}
+  {% set manifest_url = "--manifest-url=http://metadata.google.internal/computeMetadata/v1beta1/instance/attributes/google-container-manifest" -%}
```

Comment:
Ugh, that means that an internal change I have won't work (I haven't tested it yet).